schemas/list-workers-response.yml

```diff
@@ -0,0 +1,43 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              "List Workers Response"
+description: |
+  Response from a `listWorkers` request.
+type:               object
+properties:
+  workers:
+    type:           array
+    title:          "Workers from a WorkerType"
+    description: |
+      List of workers in this worker-type.
+    items:
+      type:         object
+      properties:
+          workerGroup:
+                    description: |
+                      Identifier for the worker-group within which this run started.
```

Comment:
This isn't really related to a "run".  How about "Identifier for the worker group containing this worker"